flink-clients/src/main/java/org/apache/flink/client/deployment/application/executors/EmbeddedExecutor.java

```diff
@@ -163,7 +163,7 @@ private static CompletableFuture<JobID> submitJob(
             final Time rpcTimeout) {
         checkNotNull(jobGraph);
 
-        LOG.info("Submitting Job with JobId={}.", jobGraph.getJobID());
+        LOG.info("Submitting job '{}' ({}).", jobGraph.getName(), jobGraph.getJobID());
```

Comment:
The job submission log of application mode is already redundant. Therefore I prefer to not further complicate it. 

e.g.
```
2024-05-06 20:19:20,853 [flink-akka.actor.default-dispatcher-16] INFO  org.apache.flink.client.deployment.application.executors.EmbeddedExecutor [] - Job 2214f37f6f124fadb641e22182325897 is submitted.
2024-05-06 20:19:20,854 [flink-akka.actor.default-dispatcher-16] INFO  org.apache.flink.client.deployment.application.executors.EmbeddedExecutor [] - Submitting Job with JobId=2214f37f6f124fadb641e22182325897.
2024-05-06 20:19:21,141 [flink-akka.actor.default-dispatcher-5] INFO  org.apache.flink.runtime.dispatcher.StandaloneDispatcher     [] - Received JobGraph submission 'Flink Streaming Job' (2214f37f6f124fadb641e22182325897).
2024-05-06 20:19:21,177 [flink-akka.actor.default-dispatcher-5] INFO  org.apache.flink.runtime.dispatcher.StandaloneDispatcher     [] - Submitting job 'Flink Streaming Job' (2214f37f6f124fadb641e22182325897).
```